src/app.ts

```diff
@@ -116,6 +120,13 @@ async function app() {
   // Trim empty spaces from command line dependencies
   const dependencies = programOptions.dependency?.map((dep: string) => dep.trim());
 
+  // Use specified FHIR Version
+  const specifiedFHIRVersion = programOptions.useFHIRVersion;
+  if (specifiedFHIRVersion && !utils.isSupportedFHIRVersion(specifiedFHIRVersion)) {
+    logger.error(`Specified FHIR version is invalid: ${specifiedFHIRVersion}`);
```

Comment:
Does it make sense to say "Specified FHIR version is not supported" here instead? Maybe it is a valid FHIR version but just one that we don't support?

Reply:
I agree.  `not supported` is better than `invalid`.

Reply:
Done.